.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,6 +10,8 @@
 
 <!-- Large change: Provide instructions to reviewers how to read the diff. -->
 
-# Priorities
+# Priorities and Process
 
 Add :+1: to [pull requests you find important](https://github.com/NixOS/nix/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc).
+
+The Nix maintainer team uses [this project board](https://github.com/orgs/NixOS/projects/19) to track and schedule reviews.
```

Comment:
We may want to also add a note on the process we use for that. Feel free to disagree and merge as is though:

```suggestion
The Nix maintainer team uses a [GitHub project board](https://github.com/orgs/NixOS/projects/19) to [track and schedule reviews](https://github.com/NixOS/nix/tree/master/maintainers#project-board-protocol). 
```